src/fides/api/service/storage/streaming/smart_open_streaming_storage.py

```diff
@@ -688,15 +688,15 @@ def _handle_html_format_upload(
         attachment_files_generator = self._create_attachment_files(all_attachments)
 
         # Combine both generators and stream the complete ZIP to storage
+        from itertools import chain
```

Comment:
We can keep this at the top of the file, I just put it in the snippet as an example